### ISSUE TITLE
Maze not populating correctly

### ISSUE THREAD

**Issue by Shadowedvaca** [Shadowedvaca/Meandering-Muck-2.0#18]
![image](https://github.com/Shadowedvaca/Meandering-Muck-2.0/assets/20402103/b7a37a89-c7af-43e8-9cee-1ba7d2ae6553)

**Comment by Shadowedvaca** [comment]
Now it is even worse.  Need to look into the changes made tonite and try to track it down...

**Comment by Shadowedvaca** [comment]
Never mind, I think it was when I deleted main menu and readded it to main that it broke a connect and the maze wasn't drawing.  It's back to looking like the screenshot above.

**Comment by Shadowedvaca** [comment]
Looking at the JSON, I see two things
1 - The fill isn't working.  Hypothesis = the value is cleared, x is loaded, the value is cleared, y is loaded
2 - The possible positions is wrong too, but in a different way.  Instead of only y being populated, there is only a single value in x and y
3 - Even if the possible positions were correct, tho ones it picked were legal for wall placement (2,2) is = min open.  But it settled on (-1,-1).  That calculation isn't working.

1 / 2 are a sign of problems with how I structured those data elements.  I overcomplicated it.  It should just be a pair of arrays.  I may need to go back to a min/max type of data min max script.  Else, I need to build the data element into the maze data class and the maze piece class.
